src/manifest.json

```diff
@@ -3,6 +3,10 @@
     "requires": {
         "cockpit": "125"
     },
+    "conditions": [
+        {"path-exists": "/usr/bin/rpm-ostree"},
```

Comment:
We don't strictly need this I think, but it's cheap and communicates the intention.

Reply:
Right, it's a dependency of the package already I assume, but sounds fair.

Reply:
Yes, but the point of this is to make our pages work without packaging, i.e. using them on target hosts without cockpit-bridge and any installed packages.